### PR TITLE
[#9382] Add committers

### PR DESCRIPTION
**Pull request by amrut-prabhu**
Part of #9382

Not my best photoshop work, but good enough...

2019-04-02_04h05_112019-04-02_19h11_312019-04-02_19h11_38

For reference:
2019-04-02_04h05_462019-04-02_04h06_14

### CODE REVIEW

File: src/web/data/developers.json [hunk 1]
```diff
@@ -1640,6 +1613,54 @@
     }
   ],
   "committers": [
+    {
+      "name": "Amrut Prabhu",
+      "username": "amrut-prabhu",
+      "image": "assets/images/teammembers/amrut.png",
```
Comment (AyushChatto) [inline src/web/data/developers.json]: LGTM ;)

**Comment by monmanuela** [comment]
Lmao thanks for doing this. Is there a screenshot? I wanna see lol

**Comment by AyushChatto** [comment]
@monmanuela Go to files changed, and click `View File` on the binaries

**Comment by monmanuela** [comment]
Oh nice didn't know we can do that :O is the `template.psd` being used tho?

**Comment by damithc** [comment]
Ensure that the face size of new photos matches the existing photos. yes, a screenshot of the page would be useful for the review.

**Comment by amrut-prabhu** [comment]
Added screenshots to the description

> is the template.psd being used tho?

I thought it may be useful for the future. Makes it a bit easier since all you'll have to do is insert and resize your image (without the need for any cropping to stay within the frame).
@damithc Thoughts?

2019-04-01_22h22_00

**Comment by damithc** [comment]
> @damithc Thoughts?

Previously we used https://github.com/TEAMMATES/teammates/blob/master/src/web/mockups/processed%20photo%20sample.pptx

No harm having a psd alternative.

**Comment by damithc** [comment]
Sorry, forgot to mention. If you are adding the psd file, may be put both psd and pptx files in the same place where they can be located easily?

**Comment by amrut-prabhu** [comment]
@xpdavid Okay to merge?

**Pull request merged**